utils/config.ts

```diff
@@ -104,6 +105,9 @@ const errorOrConfig: t.Validation<IConfig> = IConfig.decode({
   FF_ONLY_NATIONAL_SERVICES: fromNullable(process.env.FF_ONLY_NATIONAL_SERVICES)
     .map(_ => _.toLocaleLowerCase() === "true")
     .getOrElse(false),
+  FF_OPT_OUT_EMAIL_ENABLED: fromNullable(process.env.FF_OPT_OUT_EMAIL_ENABLED)
```

Comment:
```suggestion
  FF_OPT_IN_EMAIL_ENABLED: fromNullable(process.env.FF_OPT_IN_EMAIL_ENABLED)
```
change in [here](https://github.com/pagopa/io-functions-app/pull/179/files#diff-8f4625a1cf4cb245a2f9e780a9a17ff6a7132b8e070997144a67499d7ff793b4R86) as consequence.